pkg/oc/admin/image/verify-signature.go

```diff
@@ -405,6 +406,6 @@ func (ui *unparsedImage) Manifest() ([]byte, string, error) {
 }
 
 // Signatures is like ImageSource.GetSignatures, but the result is cached; it is OK to call this however often you need.
-func (ui *unparsedImage) Signatures() ([][]byte, error) {
+func (ui *unparsedImage) Signatures(context.Context) ([][]byte, error) {
```

Comment:
you going ot use this to pass a timeout or something?

Reply:
@deads2k this is just implementing the interface in containers/image... this func now require context because it fetches stuff from remote locations...